src/liblibc/lib.rs

```diff
@@ -6027,8 +6034,9 @@ pub mod funcs {
               target_os = "freebsd",
               target_os = "dragonfly",
               target_os = "bitrig",
-              target_os = "openbsd",
-              target_os = "nacl"))]
+              target_os = "nacl",
+              target_os = "netbsd",
+              target_os = "openbsd"))]
```

Comment:
I started to sort it and put that TODO sort to see if that was ok. It sounds from below that it'd be ok to go ahead and do the ordering/formatting change in the same commit as long as it is minor.